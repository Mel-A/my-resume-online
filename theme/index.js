const theme = {
  colors: {
    alabaster: '#FAFAFA',
    astral: '#327DA0',
    aquamarine: '#A1DAD7',
    azure: '#0080FF',
    cobalt: '#003DB3',
    gray: '#333333',
    white: '#FFFFFF'
  },
  fonts: {
    small0: '10px',
    small8: '18px',
    medium0: '20px',
    medium4: '24px',
    big0: '30px'
  }
}

export default theme