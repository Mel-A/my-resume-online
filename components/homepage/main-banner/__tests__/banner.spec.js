import { shallow, mount } from 'enzyme'
import MainBanner from '..'
import { ThemeProvider } from "styled-components"
import theme from '../../../../theme'

jest.mock('react-i18next', () => ({
  useTranslation: () => ({t: key => key})
}))

describe('main banner', () => {
  it('main banner renders correctly', () => {
    const banner = shallow(<MainBanner />)
    expect(banner).toMatchSnapshot()
    
  })
  
  it('main banner renders theme correctly', () => {
    const banner = mount(<MainBanner />, {
      wrappingComponent: ThemeProvider,
      wrappingComponentProps: {
        theme,
      },
    })
    expect(banner).toMatchSnapshot()
  })
})
