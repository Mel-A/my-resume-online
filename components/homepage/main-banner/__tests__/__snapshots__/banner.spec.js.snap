// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main banner main banner renders correctly 1`] = `ShallowWrapper {}`;

exports[`main banner main banner renders theme correctly 1`] = `ReactWrapper {}`;
